pkg/apis/pravega/v1beta1/pravega.go

```diff
@@ -68,6 +68,86 @@ const (
 
 	// DefaultInfluxDBSecretMountDir is the default mountpath of influxdb secret
 	DefaultInfluxDBSecretMountDir = "/etc/influxdb-secret-volume"
+
+	// DefaultReadinessProbeInitialDelaySeconds is the default initial delay (in seconds)
+	// for the readiness probe
+	DefaultControllerReadinessProbeInitialDelaySeconds = 20
+
+	// DefaultReadinessProbePeriodSeconds is the default probe period (in seconds)
+	// for the readiness probe
+	DefaultControllerReadinessProbePeriodSeconds = 10
+
+	// DefaultReadinessProbeFailureThreshold is the default probe failure threshold
+	// for the readiness probe
+	DefaultControllerReadinessProbeFailureThreshold = 3
+
+	// DefaultReadinessProbeSuccessThreshold is the default probe success threshold
+	// for the readiness probe
+	DefaultControllerReadinessProbeSuccessThreshold = 3
+
+	// DefaultReadinessProbeTimeoutSeconds is the default probe timeout (in seconds)
+	// for the readiness probe
+	DefaultControllerReadinessProbeTimeoutSeconds = 60
+
+	// DefaultLivenessProbeInitialDelaySeconds is the default initial delay (in seconds)
+	// for the liveness probe
+	DefaultControllerLivenessProbeInitialDelaySeconds = 60
+
+	// DefaultLivenessProbePeriodSeconds is the default probe period (in seconds)
+	// for the liveness probe
+	DefaultControllerLivenessProbePeriodSeconds = 15
+
+	// DefaultLivenessProbeFailureThreshold is the default probe failure threshold
+	// for the liveness probe
+	DefaultControllerLivenessProbeFailureThreshold = 4
+
+	// DefaultLivenessProbeSuccessThreshold is the default probe success threshold
+	// for the liveness probe
+	DefaultControllerLivenessProbeSuccessThreshold = 1
+
+	// DefaultLivenessProbeTimeoutSeconds is the default probe timeout (in seconds)
+	// for the liveness probe
+	DefaultControllerLivenessProbeTimeoutSeconds = 5
+
+	// DefaultReadinessProbeInitialDelaySeconds is the default initial delay (in seconds)
+	// for the readiness probe
+	DefaultSegmentStoreReadinessProbeInitialDelaySeconds = 10
+
+	// DefaultReadinessProbePeriodSeconds is the default probe period (in seconds)
+	// for the readiness probe
+	DefaultSegmentStoreReadinessProbePeriodSeconds = 10
+
+	// DefaultReadinessProbeFailureThreshold is the default probe failure threshold
+	// for the readiness probe
+	DefaultSegmentStoreReadinessProbeFailureThreshold = 30
+
+	// DefaultReadinessProbeSuccessThreshold is the default probe success threshold
+	// for the readiness probe
+	DefaultSegmentStoreReadinessProbeSuccessThreshold = 1
+
+	// DefaultReadinessProbeTimeoutSeconds is the default probe timeout (in seconds)
+	// for the readiness probe
+	DefaultSegmentStoreReadinessProbeTimeoutSeconds = 5
```

Comment:
The original comment for the segment store readiness probe (currently being removed from `pravega_segmentstore.go`) was that we give it up to 5 minutes to start up.
Therefore we need to have `DefaultSegmentStoreReadinessProbeTimeoutSeconds = 300`  (failure threshold of 30 times period seconds of 10) in case it is enforced by kubernetes.